engine/Core.php

```diff
@@ -205,11 +206,11 @@ private function attempt_init_child_controller($controller_path) {
         }
 
         $this->draw_error_page();
+        die(); //end of the line (all possible scenarios tried)
     }
 
     private function draw_error_page() {
         load('error_404');
-        die(); //end of the line (all possible scenarios tried)
```

Comment:
**you can't remove the die() from the draw_error_page function** as this function will give a 404 page and terminate the php script. There are multiple places used this draw_error_page function and it is a security concern!

Reply:
The core should now be unchanged from the original as I reverted it.. all I can think is this version of the core is an older version I had that hasn’t updated. Thanks will update to the latest version.

Reply:
Ah, no worries. Even tho I have suggested previously you can just edit on the same pull request, but if you don't feel it, you can close this one and open a new one.

By the way, again, **this is only my personal point of view**. At this stage, Trongate is really looking for stability as it will be released to the general public very soon. So, it expects developers to submit a PR for bug report and fix, if it is a feature request, it may get reviewed slower or even get rejected in the end. **Please, please, please** do not feel frustrated if any of those things happened.

Reply:
Hi Jake, the core should now be back to its original version. 👍 Thanks for pointing this out and the tip on updating the same pull request. 

No problem at all, I will be using it and just wanted to share in case people think this feature is useful for them. Again, apologies for doing things incorrectly, this is the first time I have attempted contributing via github. 

Keep up the great work, exciting times here 😊

Cheers,
Adam